doc/source/whatsnew/v1.0.0.rst

```diff
@@ -874,6 +874,7 @@ ExtensionArray
 - Bug in :class:`arrays.PandasArray` when setting a scalar string (:issue:`28118`, :issue:`28150`).
 - Bug where nullable integers could not be compared to strings (:issue:`28930`)
 - Bug where :class:`DataFrame` constructor raised ValueError with list-like data and ``dtype`` specified (:issue:`30280`)
+- Bug in :func:`arrays.integer.coerce_to_array` while using large-ish integers with ``NaN`` values in ``pd.Series`` (:issue:`30268`)
```

Comment:
```suggestion
- Bug in creating nullable integer Series while using large-ish integers with ``NaN`` values (:issue:`30268`)
```

(the `arrays.integer.coerce_to_array` is not public so shouldn't be mentioned in the whatsnew docs)